megatron/rl/agent/api.py

```diff
@@ -1,13 +1,14 @@
-# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 
 import asyncio
 from abc import ABC, abstractmethod
 from collections.abc import AsyncIterable
+from typing import TypeVar
 
 import numpy as np
 from pydantic import BaseModel
 
-from ..__init__ import Request, trace_async_exceptions
+from ..__init__ import Request, TypeLookupable, trace_async_exceptions
```

Comment:
Do these really need to be relative imports? Why can we not do `from megatron.rl import TypeLookupable` etc?